packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -76,13 +93,31 @@ class PaginatedDataTable extends StatefulWidget {
   final int initialFirstRowIndex;
 
   /// Invoked when the user switches to another page.
+  ///
+  /// The value is the index of the first row on the currently displayed page.
   final ValueChanged<int> onPageChanged;
 
   /// The number of rows to show on each page.
   ///
-  /// See also [onRowsPerPageChanged].
+  /// See also:
+  ///
+  /// * [onRowsPerPageChanged]
+  /// * [defaultRowsPerPage]
   final int rowsPerPage;
 
+  /// The default value for [rowsPerPage].
+  ///
+  /// Useful when initializing the field that will hold the current
+  /// [rowsPerPage], when implemented [onRowsPerPageChanged].
+  static const int defaultRowsPerPage = 10;
```

Comment:
This is on a class, so it doesn't squat on useful namespace. We've avoided `k` in cases like that before (e.g. Checkbox.width).